timber/src/main/java/timber/log/Timber.java

```diff
@@ -446,13 +455,11 @@ protected String createStackElementTag(StackTraceElement element) {
      *
      * {@inheritDoc}
      */
-    @Override protected void log(int priority, String tag, String message, Throwable t) {
+    @Override protected void log(int priority, String tag, String message, Throwable e) {
+      String callerInfo = getCallerInfo(new Throwable().getStackTrace());
```

Comment:
As I know getting stack trace is expensive operation, should we really get it twice? Maybe caching will be more preferable?